DCCpp/PacketRegister.cpp

```diff
@@ -98,7 +98,7 @@ void RegisterList::loadPacket(int nReg, byte *b, int nBytes, int nRepeat, int pr
 
 ///////////////////////////////////////////////////////////////////////////////
 
-void RegisterList::setThrottle(char *s) volatile{
+void RegisterList::setThrottle(const char *s) volatile{
```

Comment:
since you have added braces to all single line loops using a space before the {, on a subsequent PR please run a code format plugin across the codebase to standardize all code to a single format.

Reply:
Do you have a formatter in mind? The Arduino IDE has its own. I don't know what Gregg uses. There is LLVM, Google, the Defauls Visual Code one in Platform IO. But I think that will indent everything 4 spaces. Basically, I don't want to screw up the formatting ;) What standard should we use?

Reply:
I don't have a preference really, what is critical is setting a standard and sticking to it. It should be sent as a different PR since it will be challenging to review those changes in this set etc.